src/main/java/frc/robot/HardwareMap.java

```diff
@@ -29,12 +31,20 @@ public static boolean isElevatorHardwarePresent() {
 		return true;
 	}
 
+	/**
+	 * Check if climber hardware is available to the RoboRIO.
+	 * @return true if climber hardware is present
+	 */
+	public static boolean isClimberHardwarePresent() {
+		return true;
```

Comment:
Let's leave these as "false" on main branch until the mech is installed on the robot.